flow/aggregate.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+from collections.abc import Iterable
+from copy import deepcopy
+from itertools import groupby
+from itertools import zip_longest
+from functools import partial
+
+import signac
+from signac.contrib.project import Project, JobsCursor
+
+
+class aggregate:
```

Comment:
```suggestion
class Aggregate:
```

should this be capitalized because it's a class?